source/js/components/ajax.js

```diff
@@ -0,0 +1,33 @@
+import fetchUtils from "@degjs/fetch-utils";
```

Comment:
[fix] I would consider restructuring or removing this file entirely -- it's essentially a wrapper for fetchUtils that doesn't really add any value over just importing and using fetchUtils directly. I can kind of see what you're trying to do to make this a reusable plugin, but it doesn't actually work (notice that there is no way to actually pass settings into the component, or how you have a querySelector directly inside of this when you could just pass in a value). If you are looking to keep some consistent fetchUtils settings (which might be nice on a larger project), I think you'd be better suited to just make a `fetchConfig` utility function or even static JSON that you can import.